core/storage/suggestion/gae_models.py

```diff
@@ -199,11 +206,11 @@ def get_export_policy(cls):
             'target_version_at_submission':
                 base_models.EXPORT_POLICY.EXPORTED,
             'status': base_models.EXPORT_POLICY.EXPORTED,
-            'author_id': base_models.EXPORT_POLICY.EXPORTED,
-            'final_reviewer_id': base_models.EXPORT_POLICY.EXPORTED,
+            'author_id': base_models.EXPORT_POLICY.NOT_APPLICABLE,
```

Comment:
Add line comments explaining why this is "not applicable", ditto for the next field.

Reply:
done